donate/models.py

```diff
@@ -162,12 +162,12 @@ class StripeDonation(Donation, TimestampMixin):
     id = db.Column(db.Integer,
                    db.ForeignKey('donation.id'),
                    primary_key=True)
-    card_id = db.Column(db.String,
+    card_id = db.Column(db.String(64),
```

Comment:
Is there some documentation from stripe that says the card id will always be no longer than 64 characters?

Reply:
good point, it's stated to be no more than 255 in the doc.  Updated.  Thx